xcube/core/store/store.py

```diff
@@ -228,6 +228,34 @@ def get_open_data_params_schema(self, data_id: str = None, opener_id: str = None
         identified opener. Some openers might not support this, therefore *opener_id* is optional, and if
         it is omitted, the returned schema will be less restrictive.
 
+        For maximum compatibility of stores, it is strongly encouraged to apply the following conventions on
+        parameter names, types, and their interpretation.
+
+        Let P be the value of an optional, data constraining open parameter,
+        then it should be interpreted as follows:
+
+          * _if P is None_ means, parameter not given, hence no constraint applies, hence full containment.
+          * _if not P_ means, we exclude what would otherwise be fully included.
+          * _else_, the given constraint applies.
+
+        Given here are names, types, and descriptions of common, constraining open parameters for gridded datasets.
+        Note, whether any of these is optional or mandatory depends on the individual data store. A store may also
+        define other open parameters or support only a subset of the following:
+
+          * ``variable_names: List[str]``: Included data variables.
+             Available coordinate variables will be auto-included for any dimension of the data variables.
+          * ``bbox: Tuple[float, float, float, float]``: Spatial coverage.
+          * ``crs: str``: Spatial CRS, e.g. "EPSG:4326" or OGC CRS URI.
+          * ``spatial_res: float``: Spatial resolution in coordinates of the spatial CRS.
+          * ``time_range: Tuple[Optional[str], Optional[str]]``: Time range interval using iso-date/times.
```

Comment:
We should be explicit about the semantics of None in the time range (which I assume should be interpreted as "from the earliest data in the dataset" and "up to the latest data in the dataset" for start and end respectively).

Reply:
Done.